tests/BuildScriptGenerator.Tests/Node/NodePlatformTest.cs

```diff
@@ -0,0 +1,205 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using Microsoft.Oryx.BuildScriptGenerator.Node;
+using Microsoft.Oryx.Tests.Common;
+using Xunit;
+
+namespace Microsoft.Oryx.BuildScriptGenerator.Tests.Node
+{
+    public class NodePlatformTest
+    {
```

Comment:
imo .. it's cleaner to use a sample app instead of this hardcoded package.json

Reply:
Sure, removed this now. Can you take a look at this PR again?